tools/azure-sdk-tools/changelog_generics.md

```diff
@@ -0,0 +1,84 @@
+# Autorest v2 to v3
+
+
+## General Breaking changes
+
+This version uses a next-generation code generator that *might* introduce breaking changes.
```

Comment:
shouldn't be "that introduces"? At least all the begin_ prefixes for create / update are always breaking change.

Reply:
You're putting this comment in the generics from v2 to v3 :)

Reply:
This file is the complete set of changelog generics written in MD, since our packages are in MD nowadays